transport/server.cc

```diff
@@ -698,12 +698,21 @@ future<> cql_server::connection::process_request() {
             });
             auto istream = buf.get_istream();
             (void)_process_request_stage(this, istream, op, stream, seastar::ref(_client_state), tracing_requested, mem_permit)
-                    .then_wrapped([this, buf = std::move(buf), mem_permit, leave = std::move(leave)] (future<foreign_ptr<std::unique_ptr<cql_server::response>>> response_f) mutable {
+                    .then_wrapped([this, buf = std::move(buf), mem_permit, leave = std::move(leave), stream] (future<foreign_ptr<std::unique_ptr<cql_server::response>>> response_f) mutable {
                 try {
-                    write_response(response_f.get0(), std::move(mem_permit), _compression);
+                    if (response_f.failed()) {
+                        const auto message = format("request processing failed, error [{}]", response_f.get_exception());
+                        clogger.error("{}: {}", _client_state.get_remote_address(), message);
```

Comment:
This was an error before the patch, I just preserved this behaviour, this is already mentioned in the resolved conversation above.

In fact, this _is_ a Scylla bug to some extend. The problem occurs in `process_request_one` error handling part, when we call `make_something_error` with `msg`, which is greater than the limit of `64K` enforced by `response->write_string`. We could handle this better there (e.g. trim the string in `cql_server::response::write_string` to 64K instead of throwing), but this change is still needed as a measure to cope with bugs we don't see yet.